OOInteraction/src/commands/CSceneHandlerItemTest.cpp

```diff
@@ -149,6 +209,91 @@ Interaction::CommandResult* CSceneHandlerItemTest::execute(Visualization::Item*,
 		Visualization::VisualizationManager::instance().mainScene()->addTopLevelNode(newNode, 1);
 	}
 
+	for (auto id : movedNodes)
```

Comment:
Same as above, this `for` loop and the three after it have identical structure, so you should abstract them into a common function/lambda if you plan on using this test command any further.